client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -697,7 +612,7 @@ function PatientSubmissionScreen(props) {
                   <Tooltip placement="top" title="Enregistrez les données de cette prescription et complétez-la plus tard.">
                     <Button
                       onClick={() => saveSubmission()}
-                      disabled={!state.valid}
+                      disabled={!state.valid || isEditPrescription}
```

Comment:
Est-ce qu'on veut vraiment edit une prescription et la laisser dans un state invalid?

Reply:
That's not what id does :p
It DISABLES the button if the state is invalid OR we're editing a prescription.  (which is what we want for now)

Reply:
Don't we want to be able to save? If we don't want to (so we only submit right?) shouldn't we want to remove it instead? I feel that it will be misleading for the users if the button is there but always disabled when editing